README.md

```diff
@@ -41,11 +43,23 @@ DBSC offers an API for websites to control the lifetime of such keys, behind the
 
 DBSC is bound to a device with cryptographic keys that cannot be exported from the user’s device under normal circumstances, this is called device binding in the rest of this document. DBSC provides an API that servers can use to create a session bound to a device, and this session can periodically be refreshed with an optional cryptographic proof the session is still bound to the original device. At sign-in, the API informs the browser that a session starts, which triggers the key creation. It then instructs the browser that any time a request is made while that session is active, the browser should ensure the presence of certain cookies. If these cookies are not present, DBSC will hold network requests while querying the configured endpoint for updated cookies.
 
+### Definitions
+
+* `Session Credential` - The session cookie return from the service to the user's device. This session cookie represents sustained proof of user identity. It's purpose is to be used by the user's device to generate temporary access tokens so that requests from the user's device can be authenticated.
+* `Access Token` - A cookie or other token passed in a request body, url, or header which identifies the user. Must be a short lived token and is not protected by DBSC.
+
 ### Goals
 Reduce session theft by offering an alternative to long-lived cookie bearer tokens, that allows session authentication that is bound to the user's device. This makes the internet safer for users in that it is less likely their identity is abused, since malware is forced to act locally and thus becomes easier to detect and mitigate. At the same time the goal is to disrupt the cookie theft ecosystem and force it to adapt to new protections.
 
+### Motivating use cases
+The primary driving factor is dealing with the threat of Malware installed by an attacker on a user's device. By binding authentication sessions to the device, DBSC aims to disrupt the cookie theft industry since exfiltrating these cookies will no longer have any value. We think this will substantially reduce the success rate of cookie theft malware. Attackers would be forced to act locally on the device, which makes on-device detection and cleanup more effective, both for anti-virus software as well as for enterprise managed devices. [Source](https://blog.chromium.org/2024/04/fighting-cookie-theft-using-device.html)
```

Comment:
```suggestion
The primary driving factor is dealing with the threat of malware installed by an attacker on a user's device. By binding authentication sessions to the device, DBSC aims to disrupt the cookie theft industry since exfiltrating these cookies will no longer have any value. We think this will substantially reduce the success rate of cookie theft malware. Attackers would be forced to act locally on the device, which makes on-device detection and cleanup more effective, both for anti-virus software as well as for enterprise managed devices. [Source](https://blog.chromium.org/2024/04/fighting-cookie-theft-using-device.html)

A concrete example was the [Phishing campaign targets YouTube creators with cookie theft malware](https://blog.google/threat-analysis-group/phishing-campaign-targets-youtube-creators-cookie-theft-malware/) where attackers attempted to phish and install malware on YouTube creators to exfiltrate their cookies. However, even after the malware has been removed the attacker still has access to the YouTube creators accounts.

For the rest of the DBSC proposal, it is a requirement to explicitly handle the scenario where malware was installed on the user's device and has similar access to the browser or other installed applications, such as:
* Access to any and all data saved by the browser
* Direct access to call a TPM's public API
```